samples/attestation/host/host.cpp

```diff
@@ -264,6 +355,36 @@ int main(int argc, const char* argv[])
     if (enclave_b)
         terminate_enclave(enclave_b);
 
+    /* Set logging levels for OE and AZDCAP to previously set values */
```

Comment:
Is this restoring code really needed given that we're about to terminate the application anyway? Is there any logging can occur between from here till return?

Reply:
removed env variable setting logic inside code. Moved to CMakeList.txt.